make generate internal error

```
Running Datamodel Code Generator
Make sure to first run the install_dev recipe
datamodel-codegen --input catalog-rest-service/src/main/resources/json --input-file-type jsonschema --output ingestion/src/metadata/generated
Traceback (most recent call last):
  File "/home/runner/work/OpenMetadata/OpenMetadata/env/bin/datamodel-codegen", line 8, in <module>
    sys.exit(main())
  File "/home/runner/work/OpenMetadata/OpenMetadata/env/lib/python3.9/site-packages/datamodel_code_generator/__main__.py", line 432, in main
    pyproject_toml_path = root / "pyproject.toml"
TypeError: unsupported operand type(s) for /: 'tuple' and 'str'
make: *** [Makefile:53: generate] Error 1
```

--- a/ingestion/setup.py
+++ b/ingestion/setup.py
@@ -118,7 +118,7 @@ dev = {
     "twine",
 }
 test = {
-    "black",
+    "black==21.12b0",
     "isort",
     "pre-commit",
     "pylint",

